src/components/ErrorBoundary.tsx

```diff
@@ -96,10 +53,12 @@ export class ErrorBoundary extends Component<{}, ErrorBoundaryState> {
           <div id="error">
             <div className="error container">
               <br />
-              <img src="/static/images/errorpage/error.png" style={{ height: "15vh" }} />
+              <img src="/static/images/errorpage/error.png" alt="" style={{ height: "15vh" }} />
               <h2>Something went wrong!</h2>
```

Comment:
add some alt text maybe?

Reply:
ok

Reply:
um alt text should be readable text, things like "Logo of Bank of China", "Image of guy standing on stage" etc iirc